hazelcast/src/test/java/com/hazelcast/jet/core/ExecutionLifecycleTest.java

```diff
@@ -1002,24 +1004,36 @@ private void assertJobSucceeded(Job job) {
             JobResult jobResult = getJobResult(job);
             assertTrue(jobResult.isSuccessful());
             assertNull(jobResult.getFailureText());
+            assertFalse(jobResult.isUserCancelled());
         }
     }
 
     private void assertJobFailed(Job job, Throwable expected) {
+        boolean isCancelled = expected instanceof CancellationException;
```

Comment:
Shouldn't we use `CancellationByUserException` here?

Reply:
true, fixed